boost/multiprecision/uintwide_t_backend.hpp

```diff
@@ -33,6 +33,13 @@
 
   #include <math/wide_integer/uintwide_t.h>
 
+  #if defined(WIDE_INTEGER_NAMESPACE)
```

Comment:
@ckormanyos I'm curious to learn what these blocks do. Does my namespace change break the project if they are absent?

Reply:
>  learn what these blocks do. Does my namespace change break the project if they are absent?

I found, yes. When an outer namespace is defined and the regular tests and examples are compiled, a bit more stuff is/was needed to get the outer namespace resolved.

I'm not sure if my changes were absolutely correct or the best choices. but I added relevant CI job(s) and things seem OK, at least preliminarily.

Reply:
Oh dear, that's not the intent. If `WIDE_INTEGER_NAMESPACE` isn't defined, the change isn't meant to have any effect. LMK if there's anything I can do to fix that.

Reply:
Well, ... I must admit, I had to think twice about the outer namespace. If the outer namespace is used, then somehow client code needs to either use it (via `using`) or change the type-definitions/aliases. In fact, I had not initially seen some of the related nuances until I tried it out.

Nonetheless, I am conviced that the optional outer namespace adds value to the project by shielding namespace `math`. So I'm pretty sure this is a good way to go.

I have simplified the logic of the preprocessor stuff regarding this and its cycling now in CI. Should be good to go soon.